subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -33,7 +33,7 @@ static int download_client_callback(const struct download_client_evt *event)
 
 		err = download_client_file_size_get(&dlc, &file_size);
 		if (err != 0) {
-			LOG_ERR("download_client_file_size_get error %d", err);
+			LOG_DBG("download_client_file_size_get err: %d", err);
```

Comment:
Changed from `err` do `dbg` on purpose?

Reply:
yes this is logged by INF in download client